### DISCUSSION TITLE
Player Command

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1225] [Help] [answered]
Hi, how can I force a player to run a command?

**Comment by HSGamer** [comment, marked as answer]
https://javadoc.minestom.net/net/minestom/server/entity/Player.html#chat(java.lang.String)

Something like `player.chat("/gamemode creative")`
Can you try it? I haven't tested it so I don't know if it works.

**Comment by KrystilizeNevaDies** [comment]
It should be `MinecraftServer.getCommandManager().runCommand("/gamemode creative", player)`

**Reply by DevBlokJochem** [reply]
CommandManager().runCommand() isnt a thing.

**Comment by DevBlokJochem** [comment]
What is the difference between Player.chat and MinecraftServer.getCommandManager().runCommand() ?

**Comment by HSGamer** [comment]
https://javadoc.minestom.net/net/minestom/server/command/CommandManager.html#execute(net.minestom.server.command.CommandSender,java.lang.String)

Guess it's `MinecraftServer.getCommandManager().execute(player, "gamemode creative");`
